src/platform/user/profile/vet360/components/PhoneField/PhoneField.jsx

```diff
@@ -60,7 +120,8 @@ export default class PhoneField extends React.Component {
     };
   }
 
-  convertCleanDataToPayload(cleanData, fieldName) {
+  convertCleanDataToPayload(data, fieldName) {
+    const cleanData = this.convertNextValueToCleanData(data);
```

Comment:
Since we don't call `convertNextValueToCleanData()` with each change to the SchemaForm's data, we need to call it here as part of the pre-submission cleanup. This will have no ill effect on the existing form in prod.